pennylane/templates/subroutines/hilbert_schmidt.py

```diff
@@ -71,7 +71,7 @@ class HilbertSchmidt(Operation):
 
         .. code-block:: python
 
-            with qml.tape.QuantumTape(do_queue=False) as u_tape:
+            with qml.tape.QuantumTape(do_queue=False) as u_script:
                 qml.Hadamard(wires=0)
```

Comment:
I wonder if we should switch the example to use `qml.tape.QuantumScript([qml.Hadamard(0)])`.  But we can do a full docstring pass later.

Reply:
My intention was of course to only change the name, but I agree. Plus with the below comment, it's even weirder. Maybe I should revert the changes to hilbert_schmidt in general and do those in another PR since it should be more than just a name change?

Reply:
I think we should just leave the name as is for now.  Changing the name would be a breaking change.

Reply:
reverted all hilbert schmidt changes. when we're ready, it'll be as simple as git reverting [this commit](https://github.com/PennyLaneAI/pennylane/pull/3429/commits/b645650ece359d8532ee579c60c1582abeab1313)